smart-contracts/contracts/mixins/MixinGrantKeys.sol

```diff
@@ -44,6 +46,7 @@ contract MixinGrantKeys is
       }
       // Set the key Manager
       keyManagerOf[idTo] = keyManager;
+      _clearApproval(idTo);
```

Comment:
`_resetKeyManagerOf` could be changed to accept another param `address _keyManager`which would allow us to use it consistently here as well. For now, this approcah sidesteps the circular dependency issue.

Reply:
or simpler; just use `setKeyManagerOf` and pass `address(0)`

Reply:
Hmm I'm not sure we should clear on grant

I was thinking just for setKeyManager